### PR TITLE
DOC-200 | Notebooks: Update magic commands

### PR DESCRIPTION
**Pull request by nerpaula**
preview: https://deploy-preview-1303--zealous-morse-14392b.netlify.app/docs/3.10/arangograph/notebooks.html#arangograph-magic-commands

### CODE REVIEW

File: arangograph/notebooks.md [hunk 1]
```diff
@@ -115,8 +119,10 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%whichCollection` - returns the collection name that is currently selected.
 - `%createCollection collectionName` - creates a collection.
 - `%selectCollection collectionName` - selects a collection as the current collection.
+- `%useCollection collectionName` -
```
Comment (Simran-B) [inline arangograph/notebooks.md:122]: also an alias for `%selectCollection`?

File: arangograph/notebooks.md [hunk 2]
```diff
@@ -115,8 +119,10 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%whichCollection` - returns the collection name that is currently selected.
 - `%createCollection collectionName` - creates a collection.
 - `%selectCollection collectionName` - selects a collection as the current collection.
+- `%useCollection collectionName` - 
 - `%getCollection collectionName` - gets a collection for variable assignment,
   e.g. `student` = `% getCollection Student`.
+- `%createEdge` - creates an edge collection.
```
Comment (Simran-B) [inline arangograph/notebooks.md:125]: If this command actually creates an edge collection and not an edge, then its name is confusing.

File: arangograph/notebooks.md [hunk 3]
```diff
@@ -95,6 +95,8 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%whichDatabase` - returns the database name you are connected to.
 - `%createDatabase databaseName` - creates a database.
 - `%selectDatabase databaseName` - selects a database as the current database.
+- `%useDatabase databasename` - uses a database as the current database; shares
+  the same function with `%selectDatabase`.
```
Comment (Simran-B) [inline arangograph/notebooks.md:99]: In other words, it is an alias for `%selectDatabase`?

Comment (nerpaula) [reply arangograph/notebooks.md:99]: yes

File: arangograph/notebooks.md (hunk 2 above)
Comment (nerpaula) [reply arangograph/notebooks.md:126]: @cw00dw0rd can you please clarify?

Comment (cw00dw0rd) [reply arangograph/notebooks.md:126]: It adds an edge definition to a graph. This was added by Arthur, I will verify the desired use case and follow-up, maybe with a better name or different functionality.

Comment (cw00dw0rd) [reply arangograph/notebooks.md:126]: Update on this:
- Name will be adjusted to `createEdgeDefintion`
- Functions for `createEdgeCollection` and `createVertexCollection` will be added to further distinguish functionality

Comment (cw00dw0rd) [reply arangograph/notebooks.md:126]: I will come back here once I have pushed the new image, likely Thursday, to verify they have been added

Comment (nerpaula) [reply arangograph/notebooks.md:126]: Yes, please let us know once they are added. thanks 👍

File: arangograph/notebooks.md [hunk 4]
```diff
@@ -120,10 +120,12 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%createCollection collectionName` - creates a collection.
 - `%selectCollection collectionName` - selects a collection as the current collection.
 - `%useCollection collectionName` - uses the collection as the current collection;
-  shares the same function with `%selectCollection`.
+  alias for `%selectCollection`.
 - `%getCollection collectionName` - gets a collection for variable assignment,
   e.g. `student` = `% getCollection Student`.
-- `%createEdge` - creates an edge collection.    
+- `&createEdgeCollection` - creates an edge collection.
+- `&createVertexCollection` - creates a vertex collection.
```
Comment (Simran-B) [inline arangograph/notebooks.md:127]: Really ampersand instead of percent?

**Pull request merged**